cmd/tailscale/ui.go

```diff
@@ -338,6 +404,51 @@ func (d *Dismiss) Dismissed(gtx layout.Context) bool {
 	return false
 }
 
+func (ui *UI) layoutExitStatus(gtx layout.Context, state *BackendState) layout.Dimensions {
+	var bg color.NRGBA
+	var text string
+	switch state.ExitState {
+	case ExitNone:
+		return D{}
+	case ExitOffline:
+		text = "Traffic relay offline"
+		bg = rgb(0xc65835)
+	case ExitOnline:
+		text = "Relaying traffic via"
+		bg = rgb(0x338b51)
+	}
+	paint.Fill(gtx.Ops, bg)
+	return material.Clickable(gtx, &ui.openExitDialog, func(gtx C) D {
+		gtx.Constraints.Min.X = gtx.Constraints.Max.X
+		return layout.Inset{
+			Top:    unit.Dp(12),
+			Bottom: unit.Dp(12),
+			Right:  unit.Dp(24),
+			Left:   unit.Dp(24),
+		}.Layout(gtx, func(gtx C) D {
+			return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+				layout.Flexed(1, func(gtx C) D {
+					return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							lbl := material.Caption(ui.theme, text)
```

Comment:
Lets use the same type size at the IP addresses for both of these labels. It looks like they're a new, smaller type size than we use elsewhere.